packages/ui/src/components/Switch/base/SwitchBase.tsx

```diff
@@ -1,10 +1,10 @@
 import React, {useCallback, useState} from 'react';
 import cn from 'classnames';
-import {SwitchClasses, SwitchSize} from '../shared';
+import {SwitchClasses} from '../shared';
 import styles from './styles.scss';
 
 export interface SwitchBaseProps {
-  size?: SwitchSize;
+  size?: string;
```

Comment:
This property should not be `string`. It should be `SwitchSize` as long as we know which sizes exist (`'sm' | 'm' | 'l' | 'xl'`)